### ISSUE TITLE
Landing page in UI

### ISSUE THREAD

**Issue by lwrubel** [gwu-libraries/sfm-ui#209]
Come up with content for this page or use My Collections.

**Comment by rajatvij** [comment]
@lwrubel I am not sure if it makes sense but i think we can have instructions for user or link to instructions on how to use sfm and what is required before starting a harvest.

**Comment by kerchner** [comment]
As per discussion:
- Redirect `/` to My Collections (no Home page for now)
- If the user has 0 collections, display some content to help get started (recommend creating credentials first, etc.)
- Future Home view will be worked out in a new ticket.
